src/field/pokemon.ts

```diff
@@ -100,7 +100,7 @@ import { VariableMoveTypeMultiplierAttr } from "#app/data/moves/move-attrs/varia
 import { getNatureStatMultiplier } from "#app/data/nature";
 import { starterPassiveAbilities } from "#app/data/passives";
 import type { SpeciesEvolutionCondition, SpeciesFormEvolution } from "#app/data/pokemon-evolutions";
-import type { SpeciesFormChange } from "#app/data/pokemon-forms";
+import { SpeciesFormChangeLapseTeraTrigger, type SpeciesFormChange } from "#app/data/pokemon-forms";
```

Comment:
```suggestion
import { SpeciesFormChangeLapseTeraTrigger, type SpeciesFormChange } from "#app/data/pokemon-forms";
import type { SpeciesFormChange } from "#app/data/pokemon-forms";
```

Or did I miss the mark this time :D

Reply:
Mixed imports (`import { X, type Y } from Z`) are okay, it's only when all imports from the file are type imports (`import { type X, type Y } from Z`) that it should be brought outside the brackets. The new ESLint rule will catch this for us so we don't need to worry about it.